src/old-api/schema.ts

```diff
@@ -116,20 +116,15 @@ export class Schema extends SpecificationExtensionsModel<v2.AsyncAPISchemaObject
   }
 
   additionalProperties() {
-    if (typeof this._json === 'boolean') return this._json;
     const additionalProperties = this.__get('additionalProperties');
     if (typeof additionalProperties === 'boolean') return additionalProperties;
-    if (additionalProperties === undefined) return true;
-    if (additionalProperties === null) return false;
+    if (additionalProperties === undefined || additionalProperties === null) return;
     return this.__createChild(additionalProperties);
   }
 
   additionalItems() {
-    if (typeof this._json === 'boolean') return this._json;
     const additionalItems = this.__get('additionalItems');
-    if (typeof additionalItems === 'boolean') return additionalItems;
-    if (additionalItems === undefined) return true;
-    if (additionalItems === null) return false;
+    if (additionalItems === undefined || additionalItems === null) return;
```

Comment:
We have to be consistent with implementation in v1 parser - current logic in `next-major` has breaking change in `additionalItems` and `additionalProperties` methods and some templates won't work.

Reply:
https://github.com/asyncapi/parser-js/blob/master/lib/models/schema.js#L221
https://github.com/asyncapi/parser-js/blob/master/lib/models/schema.js#L231